english/light-fire/run-away/shout/more-info/more-info.md

```diff
@@ -0,0 +1,15 @@
+You ask him "What is going on? Where am I?"
+
+As you try to catch your breath from your marshmallow room escape, the boy sizes you up.
+
+"I'm trying to find this dog that I kind of lost. Well he ran away. Because I left the gate open..." he says.
+
+"You are in the forest near my town. What happened to you mister?"   
+
+Do you:
+
+[Help him find his dog] (find-dog/find-dog.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.